stac_validator.py

```diff
@@ -47,43 +47,24 @@ def __init__(self, stac_file, version="master"):
         :param stac_file: file to validate
         :param version: github tag - defaults to master
         """
-        git_tags = requests.get("https://cdn.staclint.com/versions.json")
-        if git_tags.status_code == 200:
-            stac_versions = git_tags.json()["versions"]
-        else:
-            git_tags = requests.get(
-                "https://api.github.com/repos/radiantearth/stac-spec/tags"
-            ).json()
-            stac_versions = [tag["name"] for tag in git_tags]
-
-        # cover master as well
-        if version is None:
-            version = "master"
-        stac_versions += ["master"]
-
-        if version not in stac_versions:
-            raise stac_exceptions.VersionException(
-                f"{version} is not a valid STAC version. Valid Versions are: {stac_versions}"
-            )
-
         self.stac_version = version
         self.stac_file = stac_file.strip()
         self.dirpath = ''
+
         self.fetch_specs(self.stac_version)
-        self.fpath = Path(stac_file)
-        self.message = {}
+        self.message = []
         self.status = {
             "catalogs": {"valid": 0, "invalid": 0},
             "collections": {"valid": 0, "invalid": 0},
             "items": {"valid": 0, "invalid": 0},
+            "unknown": 0,
```

Comment:
When we fail to parse or download a json, it might be hard to tell if it was an item a collection or a catalog. I'm 💯 open to change this

Reply:
Make sense to me.